src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
```

Comment:
Feel free to remove all comment before pushing, it was added just for your information)